Source/Atomic/Graphics/Renderer.h

```diff
@@ -598,6 +602,12 @@ class ATOMIC_API Renderer : public Object
     bool initialized_;
     /// Flag for views needing reset.
     bool resetViews_;
+    // ATOMIC BEGIN
+
+    /// Number of True primitives (3D geometry only.)
+    unsigned numSinglePassPrimitives_;

```

Comment:
I am not seeing this initialized?  Also, why is it duplicated on Renderer and the Graphics subsystems?